components/layout_2020/query.rs

```diff
@@ -456,31 +442,28 @@ fn process_offset_parent_query_inner(
             // to find it again.
             let offset_parent_node_tag = Tag::new(offset_parent_node_address);
             fragment_tree
-                .find(|fragment, _, containing_block| {
-                    match fragment {
-                        Fragment::Box(fragment) | Fragment::Float(fragment) => {
-                            if fragment.base.tag == Some(offset_parent_node_tag) {
-                                // Again, take the *first* associated CSS layout box.
-                                let padding_box_corner = fragment
-                                    .padding_rect()
-                                    .to_physical(fragment.style.writing_mode, containing_block)
-                                    .origin
-                                    .to_vector() +
-                                    containing_block.origin.to_vector();
-                                let padding_box_corner = padding_box_corner.to_untyped();
-                                Some(padding_box_corner)
-                            } else {
-                                None
-                            }
-                        },
-                        Fragment::AbsoluteOrFixedPositioned(_) |
-                        Fragment::Text(_) |
-                        Fragment::Image(_) |
-                        Fragment::IFrame(_) |
-                        Fragment::Positioning(_) => None,
-                    }
-                })
-                .unwrap()
+                    .find(|fragment, _, containing_block| {
```

Comment:
Is the additional indentation here deliberate?

Reply:
No, but it's very curious that `rustfmt` did not remove it. 🤔